### PR TITLE
no bug - Fix typos in the templates

### PR DESCRIPTION
**Pull request by sylvestre**

### CODE REVIEW

File: template/en/default/pages/release-notes3.html.tmpl [hunk 1]
```diff
@@ -2035,7 +2035,7 @@
 
 <p>[% terms.Bugzilla %] 3.2 has had some UI assistance from the NASA
   Human-Computer Interaction department and the new
-  <a href="https://wiki.mozilla.org/Bugzilla:UE">[% terms.Bugzilla %]
+  <a href="https://wiki.mozilla.org/Bugzilla:DUE">[% terms.Bugzilla %]
```
Comment (dklawren) [inline template/en/default/pages/release-notes3.html.tmpl]: This is intentional as it is a URL.

File: template/en/default/pages/quicksearch.html.tmpl [hunk 2]
```diff
@@ -389,10 +389,10 @@
   <a href="[% basepath FILTER none %]query.cgi?format=advanced">Advanced Search</a> form.</p>
 
 <ul class="qs_help">
-  <li>All [% terms.bugs %] reported by userA@company.com or assigned to him
+  <li>All [% terms.bugs %] reported by users@company.com or assigned to him
```
Comment (dklawren) [inline template/en/default/pages/quicksearch.html.tmpl]: This is intentional as it means User A (userA@company.com).

File: template/en/default/pages/quicksearch.html.tmpl [hunk 3]
```diff
@@ -389,10 +389,10 @@
...
     (the initial @ is a shortcut for the assignee, see the
     <a href="#shortcuts">Advanced Shortcuts</a> section above):<br>
-    <kbd>ALL @userA@company.com OR reporter:userA@company.com</kbd></li>
+    <kbd>ALL @users@company.com OR reporter:users@company.com</kbd></li>
```
Comment (dklawren) [inline template/en/default/pages/quicksearch.html.tmpl]: Same

File: template/en/default/pages/quicksearch.html.tmpl (hunk 2 above)
Comment (kyoshino) [reply template/en/default/pages/quicksearch.html.tmpl]:
```suggestion
  <li>All [% terms.bugs %] reported by userA@company.com or assigned to him
```

File: template/en/default/pages/quicksearch.html.tmpl (hunk 3 above)
Comment (kyoshino) [reply template/en/default/pages/quicksearch.html.tmpl]:
```suggestion
    <kbd>ALL @users@company.com OR reporter:userA@company.com</kbd></li>
```

File: template/en/default/pages/release-notes3.html.tmpl (hunk 1 above)
Comment (kyoshino) [reply template/en/default/pages/release-notes3.html.tmpl]:
```suggestion
  <a href="https://wiki.mozilla.org/Bugzilla:UE">[% terms.Bugzilla %]
```

**Comment by kyoshino** [comment]
I cannot commit suggestions due to a permission issue. Please rebase and merge them manually 🙂

**Comment by kyoshino** [comment]
Included these fixes in #1368.

**Pull request closed without merging**